app/src/main/java/vier_bier/de/habpanelviewer/MainActivity.java

```diff
@@ -128,40 +129,42 @@ protected void onCreate(Bundle savedInstanceState) {
         navigationView.addHeaderView(navHeader);
         navigationView.setNavigationItemSelectedListener(this);
 
-        if (getPackageManager().hasSystemFeature(PackageManager.FEATURE_CAMERA)) {
```

Comment:
This disables all camera functionality for pre Lollipop, doesn't it? The camera 1 API (class MotionDetector) should work from Android 1.